sky/packages/sky/lib/src/material/circle_avatar.dart

```diff
@@ -0,0 +1,46 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/gestures.dart';
+import 'package:flutter/painting.dart';
+import 'package:flutter/widgets.dart';
+
+import 'constants.dart';
+import 'ink_well.dart';
+import 'theme.dart';
+import 'typography.dart';
+
+class CircleAvatar extends StatelessComponent {
+  CircleAvatar({
+    Key key,
+    this.color,
+    this.letter
+  }) : super(key: key);
+
+  final Color color;
+  final String letter;
+
+  Widget build(BuildContext context) {
+    Color backgroundColor = color;
+    TextStyle textStyle = Typography.white.title;
+    if (backgroundColor == null) {
+      ThemeData themeData = Theme.of(context);
+      backgroundColor = themeData.primaryColor;
+      if (themeData.primaryColorBrightness == ThemeBrightness.light)
+        textStyle = Typography.black.title;
```

Comment:
I suppose we'll wait until this idiom crops up often enough to warrant a utility function. It seems like it should be

```color, textStyle = Theme.of(content).configure(color, textStyle)``` 

Except we don't have multiple value return. And `configure` isn't a very good name.